proof/refine/RISCV64/Bits_R.thy

```diff
@@ -105,10 +131,10 @@ lemma projectKO_user_data_device:
   by (cases ko)
      (auto simp: projectKO_opts_defs split: arch_kernel_object.splits)
 
-lemmas projectKOs[simp] =
-  projectKO_ntfn projectKO_ep projectKO_cte projectKO_tcb
+lemmas projectKOs =
```

Comment:
I made `projectKOs` `[simp]` in RISCV64 on purpose, because you pretty much always want them when a `projectKO` occurs. So I'd vote for putting the `[simp]` back in (and removing the explicit `simp add: projectKOs` where they occur in new proofs -- the old ones in RISCV should have any)